src/NServiceBus.RabbitMQ/Connection/IClusterHostSelectionStrategy.cs

```diff
@@ -1,42 +0,0 @@
-namespace EasyNetQ
```

Comment:
We have a praxis of "doing more than semver requires" by leaving any public type in for one extra major to give users a helpful obsolete message. Here is how it's done in the core:

https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/v6-obsoletes.cs

Does this make sense?